### PR TITLE
Adding landing page

### PR DESCRIPTION
**Pull request by b0llu**
## What to check
- [ ] src/Pages/LandingPage/LandingComponents/CategoryContainer/CategoryContainer.jsx
- [ ] src/Pages/LandingPage/LandingComponents/LandingHeroContainer/LandingHeroContainer.jsx
- [ ] src/Pages/LandingPage/LandingPage.jsx

## What not to check
- [ ] Do Not Check Any Other Files

Live Preview,
https://stormplay.netlify.app/

### CODE REVIEW

File: src/Components/LandingComponent/LandingContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+export const LandingContainer = ({ children }) => {
+  // const location = useLocation();
+
+  return (
+    <section
+      className=
+        // location.pathname === "/login" ||
+        // location.pathname === "/signup" ||
+        // location.pathname === "/"
+         "landing-container-auth"
```
Comment (Maianki) [inline src/Components/LandingComponent/LandingContainer.jsx:13]: you can remove the commented code.

File: src/Components/Header/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+          <div className="icon-container">
+            <p className="icon-color">username</p>
+            <div className="badge">
+              <i className="fa-solid fas fa-user icon-color"></i>
+            </div>
+            {/* <Link to="/" element={<LoginBox />}>
+              {encodedToken && (
+                <div onClick={() => signout()} className="badge">
+                  <i className="fas fa-sign-out icon-color"></i>
+                </div>
```
Comment (Maianki) [inline src/Components/Header/Header.jsx:21]: Remove the commented code.

File: src/Components/Toast/Toast.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+.toast-error {
+    position: fixed;
+    z-index: 5;
+    right: 5px;
+    top: 10rem;
+    font-size: 2rem;
+    background-color: var(--alert-color);
+    color: #000;
```
Comment (Maianki) [inline src/Components/Toast/Toast.css:19]: Here you can use CSS variables.

**Review by Maianki — APPROVED** [review]
Great work Dhruv. Your code looks good to merge.

File: src/Components/Toast/Toast.jsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+export const Toast = () => {
+  // const { forToast, dispatch } = useReducerContext();
+
+  // useEffect(() => {
+  //   const timeout = setTimeout(() => {
+  //     dispatch({ type: "TOAST_STATE_CLEAN" });
+  //   }, 1000);
+  //   return () => clearTimeout(timeout);
+  // }, [forToast.trigger]);
```
Comment (Developers-codz) [inline src/Components/Toast/Toast.jsx:13]: can remove commented code

File: src/Pages/LandingPage/LandingComponents/CategoryContainer/CategoryContainer.jsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  }, []);
+
+  return (
+    <div className="category-container">
+      <span className="category-header">Start Your Building Journey Here</span>
+      <div className="category-section">
+        {categories.map((category) => {
+          return (
+            <div key={category._id} className="category-box">
+              <img className="rsp-img" src={category.categoryImage} alt="" />
```
Comment (Developers-codz) [inline src/Pages/LandingPage/LandingComponents/CategoryContainer/CategoryContainer.jsx]: alt value is missing

**Review by Developers-codz — APPROVED** [review]
Good Folder structure and understandable code. All looks good to merge just minor observations

File: src/Components/LandingComponent/LandingContainer.jsx (hunk 1 above)
Comment (b0llu) [reply src/Components/LandingComponent/LandingContainer.jsx:13]: will do  that

File: src/Components/Toast/Toast.css (hunk 3 above)
Comment (b0llu) [reply src/Components/Toast/Toast.css:19]: will do that

File: src/Components/Header/Header.jsx (hunk 2 above)
Comment (b0llu) [reply src/Components/Header/Header.jsx:21]: will do that

File: src/Pages/LandingPage/LandingComponents/CategoryContainer/CategoryContainer.jsx (hunk 5 above)
Comment (b0llu) [reply src/Pages/LandingPage/LandingComponents/CategoryContainer/CategoryContainer.jsx]: thanks for pointing it out i will add the alt value

File: src/Components/Toast/Toast.jsx (hunk 4 above)
Comment (b0llu) [reply src/Components/Toast/Toast.jsx:13]: will do that

**Pull request merged**